src/views/projects/Issue.svelte

```diff
@@ -118,59 +210,114 @@
 <div class="issue">
   <header>
     <div class="summary">
-      <div class="summary-title txt-medium">
-        {issue.title}
-      </div>
+      {#if titleEditable}
+        <div
+          contenteditable="true"
```

Comment:
We should probably use a TextInput or Textarea component here.

Reply:
done in 56f8377